opal/util/show_help.c

```diff
@@ -178,8 +178,8 @@ static void local_delivery(const char *file, const char *topic, char *msg) {
 static void opal_show_help_error(const char *file, const char *topic) {
     char *msg;
     opal_asprintf(&msg,
-                   "%sSorry!  You were supposed to get help about:\n    %s\nfrom the file:\n  "
-                   "  %s\nBut I couldn't find that topic in the file.  Sorry!\n%s",
+                   "%sSorry!  You were supposed to get help about:\n    %s\nfrom encoded file:\n  "
```

Comment:
I think "encoded" will confuse everyone.  Maybe "from topic %s"?

Reply:
The topic is already shown in the first %s.  So it'll look something like:

```
--------------
Sorry!  You were supposed to get help about:
    stacktrace bad signal
from encoded file:
    help-opal-util.txt
...etc
```

I figured that if this happens out in the wild, the user will copy and paste the whole message to us, and having the encoded filename in the message will help us developers track down where the problem is.

Reply:
I was thinking something like:

```
Sorry!  You were supposed to get help about: 
    stacktrace bad signal
from topic:
   help-opal-util.txt
....
```

Or maybe "topic area" or "subsystem" or something?  I'm not sure encoded file is going to get people to the right place after you're retired :).

Reply:
Ok done.